Sources/HaCWebsiteLib/ViewModels/Hackathons/CountDownTimer.js

```diff
@@ -0,0 +1,38 @@
+function updateCountDownTimer() {
+    const startDate = {{startDate}};
+    const endDate = {{endDate}};
+    const CountDownTimerId = {{id}};
+    const CountDownTimerPreId = {{preId}};
+    const beforeEventMessage = {{beforeEventMessage}}
```

Comment:
Let's be consistent with semicolons

Reply:
Worth noting that we're going semicolonless in our gulpfile, following [this standard](https://github.com/standard/standard)

(up for discussion on this, can probably wait until later)